website/docs/error-reference.md

```diff
@@ -479,7 +479,7 @@ This error indicates a call to a method we believe does not exist (a la Ruby's
 
     See the [RBI](rbi.md) docs for how to regenerate the `*.rbi` files.
 
-1.  Sorbet will complain about this code:
+1.  <a id="include-kernel">Sorbet will complain about this code:</a>
```

Comment:
Heads up that if you link to this, it's going to be covered by the header:

<img width="804" alt="Screen Shot 2019-06-25 at 4 19 51 PM" src="https://user-images.githubusercontent.com/5544532/60140058-27660800-9765-11e9-9794-19f976fe6bea.png">

You can get around that like this:

```suggestion
1.  <a class="anchor" aria-hidden="true" id="include-kernel"></a>Sorbet will complain about this code:
```

Which is similar to the markup Docusaurus generates for normal heading tags.

Reply:
Thanks! I was unhappy with how the header hid the item heading.